Sofa/Component/LinearSolver/Direct/extlibs/metis-5.1.0/GKlib/GKlibSystem.cmake

```diff
@@ -19,7 +19,7 @@ option(METIS-GKLIB_PCRE "enable PCRE support" OFF)
 mark_as_advanced(METIS-GKLIB_PCRE)
 option(METIS-GKLIB_GKREGEX "enable GKREGEX support" OFF)
 mark_as_advanced(METIS-GKLIB_GKREGEX)
-option(METIS-GKLIB_GKRAND "enable GKRAND support" OFF)
+option(METIS-GKLIB_GKRAND "enable GKRAND support. GKRAND support allows the portability of random number generation across different architectures" ON)
```

Comment:
I suggest to `set(METIS-GKLIB_GKRAND ON)` in the CMakeLists of LinearSolver.Direct (before adding the extlib) instead of modifying the extlib.

Reply:
It makes sense. Thanks for the review